integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -99,6 +99,11 @@ master:
     web: 19999
   hostPID: false
   hostNetwork: false
+  shareProcessNamespace: false
+  extraContainers: []
+  extraVolumeMounts: []
+  extraVolumes: []
+  extraServicePorts: []
```

Comment:
nit: To maximize backward compatibility, you might wanna make these fields optional. I mean you can add these fields to `values.yaml` but some users might just overwrite the `values.yaml` with their old ones, therefore missing these fields. If you add an `if .Values.blah` check to each of these fields in your `helpers.tpl`, that might be much safer.

However, I don't think that is a must-have. During the development of helm chart we do add new fields just like this and assume users will not screw with the `values.yaml`. The current implementation is fine. Pls treat this just as an additional mental guard if you are to apply more changes.

Reply:
ah thanks, i'll add the checks to prevent potential future issues